packages/shared/world/TeleportController.ts

```diff
@@ -90,6 +113,17 @@ export class TeleportController {
   }
 }
 
+async function fetchHomePoint(): Promise<{ x: number; y: number }> {
+  const homePoint: string = await getFromPersistentStorage(homePointKey)
+  if (homePoint) {
+    store.dispatch(sendHomeScene(homePoint))
```

Comment:
All code looks good but I don't find a reason for this line
```suggestion
    store.dispatch(sendHomeScene(homePoint))
```

Reply:
I thought this line was required to send the scene to the unity client, if there is no point in it, let's delete it. Thanks.